concepts/match-basics/about.md

```diff
@@ -29,18 +9,46 @@ enum Color {
     Green,
 }
 
-fn return_color_type(color: Color) -> felt252 {
+fn print_color_type(color: Color) {
     match color {
-        Color:Red => println!("The color is Red"),`
-        Color:Yellow => println!("The color is Yellow"),
-        Color:Green => println!("The color is Green"),
+        Color::Red => println!("The color is Red"),
+        Color::Yellow => println!("The color is Yellow"),
+        Color::Green => println!("The color is Green"),
     }
 }
-```
 
-The `return_color_type` function prints the color of whichever unknown parameter was provided. Given the unknown `color` value, it would have several patterns that involves each of the variants to match. The arm of code to execute for each pattern is a one liner `println!("The color is ${color}")` of whatever the value match. To handle mutiple lines of code, the arm of code should be wrapped into a curly braces `{}`.
+// To handle multiple lines of code, the `match` arm of code should be wrapped in curly braces `{}`:
```

Comment:
I think it would look better if the multiline example was separate from the single-line example.
- place this sentence below the code block as a regular sentence, and move the multi-line example below it
- reuse the `specified_colour` function from https://starknet-by-example.voyager.online/getting-started/cairo_cheatsheet/match.html?highlight=match#match, as it makes a bit more sense than the `print_color_name` one below.